src/databases/CGNS/avtCGNSFileReader.C

```diff
@@ -1625,6 +1629,151 @@ avtCGNSFileReader::GetCoords(int timestate, int base, int zone, const cgsize_t *
     return !err;
 }
 
+// ****************************************************************************
+// Method: avtCGNSFileReader::GetQuadGhostZones
+//
+// Purpose:
+//   Add the ghost zone information to the specified data set.
+//
+// Arguments:
+//   base       : The base to use
+//   zone       : The zone (mesh) that whose rind data we want.
+//   zsize      : The zone size information.
+//   cell_dim   : The cell dimension.
+//   ds         : The data set to add the ghost data to.
+//
+// Programmer: Eric Brugger
+// Creation:   Tue Jul  6 10:27:03 PDT 2021
+//
+// Modifications:
+//
+// ****************************************************************************
+
+void
+avtCGNSFileReader::GetQuadGhostZones(int base, int zone,
+    const cgsize_t *zsize, int cell_dim, vtkDataSet *ds)
+{
+    const char *mName = "avtCGNSFileReader::GetQuadGhostZones: ";
+
+    // Read the rind data.
+    debug4 << "Reading rind node." << endl;
+    int rind[6];
+    if(cg_goto(GetFileHandle(), base, "Zone_t", zone, "FlowSolution_t", 1, "end") == CG_OK)
+    {
+        if (cg_rind_read(rind) == CG_OK)
+        {
+            debug4 << "rind=" << rind[0] << "," << rind[1] << ","
+                              << rind[2] << "," << rind[3] << ","
+                              << rind[4] << "," << rind[5] << endl;
+        }
+        else
+        {
+            debug4 << "No rind data." << endl;
+            return;
+        }
+    }
+    else
+    {
+        debug4 << "Error going to FlowSolution." << endl;
```

Comment:
Is `FlowSolution` terminology generic to CGNS or specific to a particular kind of CGNS data?

Reply:
It is generic to all CGNS files. It is the portion of the file that contains the variables.

Reply:
It is generic to all CGNS files. It is the portion of the file that contains the variables.